src/GraphQL.NewtonsoftJson/ROMJsonConverter.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using GraphQLParser;
+using Newtonsoft.Json;
+
+namespace GraphQL.NewtonsoftJson
+{
+    /// <summary>
+    /// A custom JsonConverter for reading or writing a <see cref="ROM"/> object.
+    /// </summary>
+    public class ROMJsonConverter : JsonConverter
```

Comment:
Both ROM converters from NSJ and STJ are for writing ROMs. I added reading just in case.